ansible/www-standalone/resources/scripts/build-site.sh

```diff
@@ -44,6 +44,7 @@ docker run \
   -v /home/nodejs/.npm:/npm/ \
   node:latest \
   bash -c " \
+    apt-get update && apt-get install -y rsync && \
```

Comment:
this is a holdover from some website publish debugging we were doing a few months ago, changed on server but not in ansible, I'm unsure if this is necessary but it's there for now!